kong/plugins/datadog/handler.lua

```diff
@@ -46,6 +51,9 @@ local function log(premature, conf, message)
     if metric == "request_size" then
       request_size_gauge(api_name, message, logger)
     end
+    if metric == "response_size" then
+      response_size_gauge(api_name, message, logger)
+    end
```

Comment:
Side note: it would be more efficient to store all those methods in a table instead of doing all those `if`. Like:

```lua
local gauges = {
  request_size = function(api_name, msg, logger)

  end,
  status_count = function(api_name, msg, logger)

  end
  -- etc...
}
```
In the loop:

```lua
for _, metric in pairs(conf.metrics) do
  local gauge = gauges[metric]
  if gauge ~= nil then
    gauge(api_name, message, logger)
  end
end
```